packages/frontend/core/src/blocksuite/extensions/editor-config/toolbar/index.ts

```diff
@@ -1090,5 +1216,12 @@ export const createCustomToolbarExtension = (
         when: ctx => ctx.getSurfaceModels().length === 1,
       },
     }),
+
+    ToolbarModuleExtension({
+      id: BlockFlavourIdentifier(
+        `custom:${SurfaceRefBlockSchema.model.flavour}`
+      ),
+      config: createSurfaceRefToolbarConfig(baseUrl),
+    }),
```

Comment:
add `when: ctx => ctx.getSurfaceModels().length === 1,`, show only if only one

Reply:
I think the `surface-ref-block` is only rendered in page mode, so this constraint is unnecessary